spec/grape/middleware/base_spec.rb

```diff
@@ -76,6 +76,100 @@
     end
   end
 
+  describe '#header' do
+    class HeaderExampleWare < Grape::Middleware::Base
```

Comment:
These classes are globally scoped so it's a bit annoying in specs. You can fix this by doing 

```ruby
let(:header_example_ware) { Class.new(Grape::Middleware::base) do ... }
```